Mods/Pet_Default_v5_Dark.theme

```diff
@@ -1,20 +1,24 @@
 @description Default v5 Dark Theme
-@version 1.0
+@version 1.1
 @changelog
   - Updated to v5.22 (added preview)
+v1.1
+  - MIDI Editor Colormap reworked
+  - Changes in the ME Tracklist
+  - Separators added  
 @provides
   Default_5_Dark.ReaperThemeZip http://stash.reaper.fm/29513/Default_5_Dark.ReaperThemeZip
```

Comment:
I've decided to make two themes. This is closer to my original intention I had when I started modifying it. I created a new file here, I hope I didn't screw something. Thank you for helping me!